packages/dates/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@shopify/dates",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "Allows React apps to easily implement date checking",
```

Comment:
Something like, "Lightweight date operations for React".  We produced this code to avoid importing large libraries like `moment`.

"Lightweight".  Developers always regret saying that 😄